docs/source/getting-started-client-mode.md

```diff
@@ -0,0 +1,98 @@
+## Enabling Client Mode/Standalone Support
+
+Even if you are not leveraging the full distributed capabilities of Jupyter Enterprise Gateway, the client mode can
+still help mitigate the resource estarvation by enabling pseudo-distributed mode where kernels are started in client
+mode in different nodes of the cluster utilizing a round-robin algorithm. In this case, you can still get bottlenecks
+in a given node that might get requests to start "large" kernels, but otherwise, you will be better of compared to when
+all kernels are stsarted from a single node or as local processes which is the default for vanilla Jupyter Notebook.
+
```

Comment:
Typos: 
`estarvation` -> `starvation`
`better of compared` -> `better off compared`
`stsarted` -> `started`

Suggested phrasing:
`Even if you are not leveraging the full distributed capabilities of Jupyter Enterprise Gateway, client mode can still help mitigate resource starvation by enabling a pseudo-distributed mode, where kernels are started ion different nodes of the cluster utilizing a round-robin algorithm. In this case, you can still experience bottlenecks on a given node that receives requests to start "large" kernels, but otherwise, you will be better off compared to when all kernels are started on a single node or as local processes, which is the default for vanilla Jupyter Notebook.`